gwswlib/importer.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+import logging
+import os
+import csv
```

Comment:
Opmerking voor mezelf: ik moet isort nog toevoegen om de imports automatisch alfabetisch te sorteren. Of uitvogelen hoe ik black zover kan krijgen :-)